### PR TITLE
gnome-ssh-askpass: update to 7.5p1

### PR DESCRIPTION
**Pull request by damadmai**
Change URL from ftp to https
Add vlicense()
Add symlink for virt-manager

virt-manager looks for /usr/libexec/ssh-askpass

### CODE REVIEW

**Comment by damadmai** [comment]
I'm not absolutely sure if this way for creating the symlink is correct.

File: srcpkgs/gnome-ssh-askpass/template [hunk 1]
```diff
@@ -9,14 +9,18 @@ short_desc="GTK SSH passphrase grabber (from OpenSSH)"
...
 	cd contrib
-	${CC} ${CFLAGS} $(pkg-config --cflags gtk+-2.0) gnome-ssh-askpass2.c -o gnome-ssh-askpass $(pkg-config --libs gtk+-2.0) ${LDFLAGS} -lX11
+	${CC} ${CFLAGS} $(pkg-config --cflags gtk+-2.0) ${pkgname}2.c -o ${pkgname} $(pkg-config --libs gtk+-2.0) ${LDFLAGS} -lX11
 }
 
 do_install() {
-	vbin contrib/gnome-ssh-askpass
+	vbin contrib/${pkgname}
+	if [ ! -e /usr/libexec/ssh-askpass ]; then
+		ln -sf /usr/bin/${pkgname} /usr/libexec/ssh-askpass
```
Comment (Gottox) [inline srcpkgs/gnome-ssh-askpass/template]: the package gets installed to `$DESTDIR`

**Comment by damadmai** [comment]
Added `${DESTDIR}` at file existance check & symlink creation

File: srcpkgs/gnome-ssh-askpass/template [hunk 2]
```diff
@@ -9,14 +9,18 @@ short_desc="GTK SSH passphrase grabber (from OpenSSH)"
...
 	cd contrib
-	${CC} ${CFLAGS} $(pkg-config --cflags gtk+-2.0) gnome-ssh-askpass2.c -o gnome-ssh-askpass $(pkg-config --libs gtk+-2.0) ${LDFLAGS} -lX11
+	${CC} ${CFLAGS} $(pkg-config --cflags gtk+-2.0) ${pkgname}2.c -o ${pkgname} $(pkg-config --libs gtk+-2.0) ${LDFLAGS} -lX11
 }
 
 do_install() {
-	vbin contrib/gnome-ssh-askpass
+	vbin contrib/${pkgname}
+	if [ ! -e ${DESTDIR}/usr/libexec/ssh-askpass ]; then
+		ln -sf ${DESTDIR}/usr/bin/${pkgname} /usr/libexec/ssh-askpass
```
Comment (Gottox) [inline srcpkgs/gnome-ssh-askpass/template]: hmm... can you use `ln -sfr ${DESTDIR}/usr/bin/${pkgname} $DESTDIR/usr/libexec/ssh-askpass` instead? This way the link won't break if you mount / somewhere else.

Comment (damadmai) [inline srcpkgs/gnome-ssh-askpass/template]: Changed it, but the build fails now with:
`ln: failed to create symbolic link '/destdir//gnome-ssh-askpass-7.5p1/usr/libexec/ssh-askpass': No such file or directory`
Also not the two `//` after `destdir`

**Comment by chneukirchen** [comment]
I think the askpass selection should be done with an alternatives.

**Comment by damadmai** [comment]
You are right, that seems to be a better way to achieve that. I implemented it now with alternatives.

File: srcpkgs/lxqt-openssh-askpass/template [hunk 3]
```diff
@@ -13,3 +13,7 @@ license="LGPL-2.1"
 homepage="http://lxqt.org/"
 distfiles="https://github.com/lxde/lxqt-openssh-askpass/releases/download/${version}/lxqt-openssh-askpass-${version}.tar.xz"
 checksum=52c29549bc1ecaf3e3843e08460f1ec8f1a62bc423a46034b8189fee01803eb9
+
+alternatives="
+ ssh-askpass:/usr/libexec/ssh-askpass:/usr/bin/lxqt-openssh-askpass
+"
```
Comment (Gottox) [inline srcpkgs/lxqt-openssh-askpass/template]: increase revision

Comment (damadmai) [inline srcpkgs/lxqt-openssh-askpass/template]: done

Comment (Gottox) [inline srcpkgs/lxqt-openssh-askpass/template]: Sorry for the long delay.

Increase the revision of lxqt-openssh-askpass, not gnome-ssh-askpass.

**Pull request merged**